translations/Turkish/C/Cage/I Never Knew You/I Never Knew You

```diff
@@ -45,3 +45,7 @@ Ama kaçabilmenin tek yolu benim elim ve kollarım
 İlk defa yüz yüzeyiz, şimdi hissediyorsun benim hissettiklerimi
 Bencilce, artık nefes alamıyorsun…
 …benim yüzümden
+
+
+___________________________
+Translated by  @GuerillaFam
```

Comment:
May I ask if it's a social media handle? I certainly would love authors of translated texts to be able to receive communication from other people and be credited within each file, but we wouldn't be able to put links or handles. Otherwise it'll be considered promotion or advertisement and that may put the whole project in danger.

`Translated by` just got introduced, and we have to experiment to see what can be used to make it consistent and not break any legal boundaries related to fair use laws.

Since authors in the open-source world usually get credited like so: `Name or Nickname <and.sometimes@email>`, same format should be okay for this section. That way it'll be as close as possible to git log, plus possible to parse.

If that's the handle of person, I believe not using he `@` would make it more readable and more consistent with how authors usually get mentioned in CONTRIBUTORS.txt and other such files.